urbanopt_des/urbanopt_results.py

```diff
@@ -829,3 +1029,35 @@ def get_urbanopt_default_feature_report(self, search_dir: Path):
             return report
         else:
             raise Exception(f"Could not find default_feature_report.csv in {search_dir}")
+
+    def get_urbanopt_export_building_loads(self, search_dir: Path) -> pd.DataFrame:
+        """Return the building_loads.csv file path.
+
+        Args:
+            search_dir (Path): Path for where to start looking for the file
+
+        Returns:
+            dict: dictionary of the default_feature_report.json file
+        """
+        report_file = self._search_for_file_in_reports(search_dir, "building_loads.csv", measure_name="export_modelica_loads")
+        print(f"Processing building loads from {report_file}")
+        if report_file.exists():
```

Comment:
Super dumb nit: The `print` should probably be inside the if, no?

Reply:
yee